app/src/main/java/com/example/songme/utils/Constants.kt

```diff
@@ -15,4 +15,23 @@ object Constants {
     const val FLAG_REMOTE = 0
     const val FLAG_LOCAL = 1
     const val TAG_ACTION_BOTTOM = "ActionBottomDialog"
+    const val STORAGE_PERMISSION_CODE = 100
+    const val TITLE_PERMISSION = "Permission needed"
+    const val MESSAGE_PERMISSION = "This permission is a recommendation to play music from local"
+    const val OPEN_DIALOG_PERMISSION = "Open"
+    const val CANCEL_DIALOG_PERMISSION = "Cancel"
+    const val PERMISSION_GRANTED = "Permission granted"
+    const val PERMISSION_DENIED = "Permission denied"
+    const val MESSSAGE_ERROR_CONNECTION = "Connection error"
+    const val CONTENT = "content"
+    const val MEDIA = "media"
+    const val EXTERNAL = "external"
+    const val AUDIO = "audio"
+    const val ALBUMART = "albumart"
+    const val KIND_TRACK = "Track"
+    const val CREATE_AT = ""
+    const val LAST_MODIfIED = ""
+    const val TAG_LIST = ""
+    const val GENRE = ""
+    const val URI = ""
```

Comment:
class này có vẻ đang phình to quá. em nên tách ra nhé, có thể theo màn hình hay chức năng gì đó cho hợp lý